recipes/cotila/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
```

3.8 is the earliest that hass `cxx_std_17`: https://cmake.org/cmake/help/v3.8/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html
```suggestion
cmake_minimum_required(VERSION 3.8)
```